--- apollon-1.0.2.1/apollon/firstrun.ui.h.old	2005-07-25 11:25:30.000000000 +0200
+++ apollon-1.0.2.1/apollon/firstrun.ui.h	2005-07-25 11:28:07.000000000 +0200
@@ -50,13 +50,13 @@
 	{
 		
 		//where are the plugins installed?
-		QDir installedPath ("/usr/share/giFT/giftd.conf.template");
-		if (QFile::exists(installedPath.path())) {
-			installedPath = "/usr/share/giFT/";
+		QDir installedPath;
+		if (QFile::exists("/usr/share/giFT/giftd.conf.template")) {
+			installedPath = "/usr/share/giFT";
 		}
 		else {
 			installedPath = "/usr/local/share/giFT/giftd.conf.template";
-			if (QFile::exists(installedPath.path()))
+			if (QFile::exists("/usr/local/share/giFT/giftd.conf.template"))
 				installedPath = "/usr/local/share/giFT";
 			else {
 				KMessageBox::sorry(this, 
